### PR TITLE
Adding multiple covariate options

### PR DESCRIPTION
**Pull request by TomMaullin**
This PR addresses issue #25 by adding options to specify multiple regressors at once, either using a file containing a structure or two files, one containing a matrix and one containing a list of filenames (a previous attempt of this can be found here: #44 ).

### CODE REVIEW

**Comment by TomMaullin** [comment]
Hi @nicholst ,

Apologies for the delay in submitting this PR. I started it from scratch after our meeting on Monday (the old PR is here: #44 ) so that it now mimics the layout of `spm_cfg_factorial_design` as requested. The tests have been updated to account for the new batch options and all pass and this PR is now ready for review! Please let me know if you have any feedback!

File: swe_cfg_design.m [hunk 1]
```diff
@@ -159,7 +119,42 @@
...
+% multi_reg Multiple covariates
+%--------------------------------------------------------------------------
+cov         = cfg_files;
+cov.tag     = 'files';
+cov.name    = 'File(s)';
+cov.val     = {{''}};
+cov.help    = {
+               'Select the *.mat/*.txt file(s) containing details of your multiple covariates. '
+               ''
+               'A *.mat file must contain a matrix R, and may optionally contain a cell array called ''names'' with covariate names. A *.txt file can be any file that can be read by Matlab''s IMPORTDATA function, which includes a data-only text file, with spaces, tabs or comma separators. A header row is allowed, and if present will be used to obtain the covariate names. If names for the covariates are not give they will be named R1, R2, R3, ..etc.'
```
Comment (nicholst) [inline swe_cfg_design.m]: "are not give they": give -> given

**Comment by nicholst** [comment]
Looks good! Merge away!

**Comment by nicholst** [comment]
(I mean, after fixing the typo)

**Pull request merged**